### PR TITLE
Update gems and fix CI

### PR DESCRIPTION
**Pull request by dfed**
This PR will make @dependabot happier. It was created by running `bundle update`. Other changes are highlighted below.

While I was here I also fixed CI

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-source 'https://rubygems.org' do
```
Comment (dfed) [inline Gemfile:1]: this style of sourcing within a `do/end` is no longer considered a best practice. Instead, one `source` per file is preferred these days.

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
-  gem 'cocoapods', '~> 1.9'
-end
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.11.0'
```
Comment (dfed) [inline Gemfile:3]: 1.11.0 is the latest

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   xcode-build:
     name: Xcode Build
-    runs-on: macOS-latest
```
Comment (dfed) [inline .github/workflows/ci.yml:12]: `macOS-latest` does not have Xcode 11.7 anymore

**Review by NickEntin — APPROVED** [review]
Thanks Dan!

**Pull request merged**